spec/easyrsa/03_ca_spec.rb

```diff
@@ -1,7 +1,7 @@
 require File.join(File.dirname(__FILE__), '..', 'spec_helper')
 
 describe EasyRSA::CA, 'Should' do
-  include_context "shared environment"
+  include_context 'shared environment'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.